tads/osportable.cc

```diff
@@ -598,14 +714,13 @@ os_file_stat( const char *fname, int follow_links, os_file_stat_t *s )
     s->mode = buf.st_mode;
     s->attrs = 0;
 
+#ifdef _WIN32
+    s->attrs = oss_get_file_attrs(fname);
+#else ifndef _WIN32
```

Comment:
This should just be ``#else`` (as written the ``ifndef _WIN32`` is extraneous and technically illegal though most compilers will ignore it).